server/main.go

```diff
@@ -192,6 +209,8 @@ func Server(resourceService interfaces.IService, oauthService *oauthSvc.Service)
 				Ctl.ClientProfileHandler(w, r)
 			case path == "/api/v1/verify-email":
 				Ctl.VerifyEmailHandler(w, r)
+			case path == "/api/v1/verify-code":
```

Comment:
what does verify-code means? is it for email? or any other type of code? lets specify the path to be more clear